packages/commerce-sdk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commerce-sdk-react-preview",
```

Comment:
@kevinxh I wonder if you want to include in the PR updating the package names to include the org.

And in this package name specifically remove `-preview`.